.operations/CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+## Contribution model
+
+### Steering committee 🏆
+
+Members of the steering committee work together to provide guidance and future direction to the project. Each committee member has a particular expertise which they share their knowledge on, and work to lead further improvements to the project in that area. In addition, each member of the committee leads a project tracked under our Github projects.
+
+### Collaborators 👍
+
+Collaborators are members who are contributing to the repository on a regular basis, through suggesting new best practices, triaging issues, reviewing pull requests and more. They help the steering committee to ensure that the content provided is of a high standard, up-to-date with the industry, and available in many languages. Members who are displaying the above activity, and quality of contribution will be invited to become a collaborator.
+
+On the occasion where a collaborator becomes inactive for some time, they will be asked if they would still like to contribute as a collaborator or leave their role, being acknowledged as a past collaborator.
```

Comment:
nit: Rephrase to better reflect the process. Add a missing clause for past collaborators to be able to request to be restored to active status.

```suggestion
The steering committee periodically reviews the collaborator list to identify inactive collaborators. Inactive collaborators can choose to either continue in or step down from their role, in which case they are acknowledged as a past collaborator. They may later request that the steering committee restore them to active status.
```